### ISSUE TITLE
Opening new lines for every character typed

### ISSUE THREAD

**Issue by loremipson** [oh-my-fish/oh-my-fish#534]
Operating System: macOS Sierra
Fish Version: 2.6.0
Oh My Fish Version: 5
Git version: 2.13.0
Iterm2 version: 3.0.15

![omf](https://user-images.githubusercontent.com/649890/28836926-3002f6f6-76a8-11e7-81b5-33adf203d60b.gif)

Strange behavior when typing. This is the default behavior I'm experiencing as soon as I run the `curl -L https://get.oh-my.fish | fish` command and omf install completes. I've tried various different themes and all of them are behaving this same way.

Curious if there's already a known work around for this one.

Thanks

**Comment by oranja** [comment]
This is a very strange and interesting issue.
The only thing that comes to mind is maybe you have your `$TERM` variable defined in some funky way, or you have some `fish` function that responds to every keystroke somehow?
Do you get the same behaviour in other terminal emulators? (e.g.: `Terminal.app` or `Terminator`?)

**Comment by bobthecow** [comment]
Do you have the "ambiguous character" width thing checked in settings?

**Comment by loremipson** [comment]
Thanks for replying @oranja and @bobthecow.

I checked into the ambiguous characters and my `$TERM` variable but they were seemingly fine.

I did notice that `terminal.app` actually was working fine, so I wiped out my iTerm2 settings and changed back to `fish` and now it seems to be working just fine. So.. I unfortunately haven't pinpointed the exact setting, but it appears to have been something in iTerm in the end. Still kind of strange though, that it wasn't a persisting issue with both zsh or shell.
